src/main/java/uk/gov/hmcts/darts/task/service/impl/AdminAutomatedTasksServiceImpl.java

```diff
@@ -101,7 +102,9 @@ private boolean shouldIncludeAutomatedTask(AutomatedTaskEntity automatedTaskEnti
         if (automatedTaskName == null || automatedTaskName.getConditionalOnSpEL() == null) {
             return true;
         }
-        return Boolean.parseBoolean(configurableBeanFactory.resolveEmbeddedValue(automatedTaskName.getConditionalOnSpEL()));
+        String embeddedValue = configurableBeanFactory.resolveEmbeddedValue(automatedTaskName.getConditionalOnSpEL());
+        assert embeddedValue != null;
```

Comment:
It's unusual to see usage of `assert`. I don't think anyone runs with assertions enabled during local dev, and they won't be enabled in any environments.

If we really think `embeddedValue` can be null, can we handle it with an exception or otherwise?

Reply:
Updated :)